mindjunkies/courses/models.py

```
@@ -138,6 +138,7 @@ def save(self, *args, **kwargs):
 
 
 class Enrollment(BaseModel):
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)